testing/simapp/simd/cmd/root.go

```diff
@@ -6,6 +6,8 @@ import (
 	"os"
 	"path/filepath"
 
+	tmcfg "github.com/tendermint/tendermint/config"
+
```

Comment:
Can you please remove the empty line here? We like to group SDK/Tendermint imports together.

Reply:
yep